src/applications/vaos/services/appointment/transformers.js

```diff
@@ -433,44 +433,46 @@ function setContained(appt) {
     }
     case APPOINTMENT_TYPES.ccRequest: {
       let contained = createPatientResourceFromRequest(appt);
-      appt.ccAppointmentRequest.preferredProviders.forEach(
-        (provider, index) => {
-          const address = [];
-          if (provider.address) {
-            address.push({
-              line: [provider.address?.street],
-              city: provider.address?.city,
-              state: provider.address?.state,
-              postalCode: provider.address?.zipCode,
-            });
-          }
-
-          contained = {
-            ...contained,
-            practitioner: {
-              id: `cc-practitioner-${appt.id}-${index}`,
-              name: provider.lastName
-                ? {
-                    text: `${provider.firstName} ${provider.lastName}`,
-                    family: provider.lastName,
-                    given: provider.firstName,
-                  }
-                : null,
-              address: provider.address ? address : null,
-              practitionerRole: [
-                {
-                  location: [
-                    {
-                      reference: `Location/cc-location-${appt.id}-${index}`,
-                      display: provider.practiceName,
-                    },
-                  ],
-                },
-              ],
-            },
-          };
-        },
-      );
+      if (appt.ccAppointmentRequest?.preferredProviders) {
```

Comment:
@vbahinwillit This might impact your PR slightly, but all I did was wrap this in an if statement.